docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/terraform-setup.md

```diff
@@ -0,0 +1,173 @@
+
+The [Camunda provided module](https://github.com/camunda/camunda-tf-eks-module) is public and can be inspected further before using it.
+
+1. In the folder where your `config.tf` relies, create an additional `cluster.tf`
+2. Paste the following contents in there to make use of the provided module
+
+```hcl
+module "eks_cluster" {
+  source = "github.com/camunda/camunda-tf-eks-module"
+
+  region  = "eu-central-1" # change to your AWS region
+  name    = "cluster-name" # change to name of your choosing
+
+  # Set CIDR ranges or use the defaults
+  cluster_service_ipv4_cidr = "10.190.0.0/16"
+  cluster_node_ipv4_cidr    = "10.192.0.0/16"
+
+  # Please supply your own secret values
+  postgresql_username = "secret_user"
+  postgresql_password = "secretvalue%23"
+}
+```
+
+There are various other input options to customize the cluster setup further.
+Those can be read up further in the [module documentation](https://github.com/camunda/camunda-tf-eks-module).
+
+In regards to secret management in Terraform, we recommend [injecting those via Vault](https://developer.hashicorp.com/terraform/tutorials/secrets/secrets-vault).
+
+### Execution
+
+1. Open a console in the created Terraform folder where `config.tf` and `cluster.tf` are
+2. Initialize the working directory
+
+```hcl
+terraform init
+```
+
+3. Apply the configuration files
+
+```hcl
+terraform apply
+```
+
+4. After checking the plan you can type `yes` to confirm the apply
+
+This will result in the creation of the EKS cluster and required configurations. This may take 20-30 minutes to be fully done.
+
+## IAM Access management
+
+The default access is that the user creating the EKS cluster has admin access. To allow others to access as well, we have to adjust the
+`aws-auth` configmap. This can either be done manually via `kubectl` or alternatively via `terraform`.
+
+Users can generate access to the EKS cluster via the `AWS CLI`.
+
+```shell
+aws eks --region <region> update-kubeconfig --name <clusterName>
+```
+
+### terraform
+
+For `terraform` you can create an iam user to Kubernetes role mapping via the following variable.
+
+```hcl
+# this is targeting IAM roles
+aws_auth_roles = [{
+    rolearn  = "<arn>"
+    username = "<username>"
+    groups   = ["system:masters"]
+  }]
+
+# this is targeting IAM users
+aws_auth_users = [{
+    userarn  = "<arn>"
+    username = "<username>"
+    groups   = ["system:masters"]
+  }]
+```
+
+Where `arn` is the `arn` of your user or the role. The `group` is the Kubernetes rule, where `system:masters` is equivalent to an admin role. Lastly `username` is either the username itself or the role name, which is used for logs.
+
+### kubectl
+
+The same can also be achieved by using `kubectl` and manually adding the mapping as part of the `mapRoles` or `mapUsers` section. This requires you to already have access to the Kubernetes cluster. We do recommend using Terraform to configure the cluster access.
```

Comment:
What is your motivation for including this alternative way to achieve the same as in the previous paragraph? This document is aiming for a quick-start guide instead of a comprehensive list of all options and considerations - so to me it feels redundant and could be confusing, especially since using Terraform is recommended.

Reply:
good point, will drop it. Users should resolve to using Terraform.